frontend/public/components/utils/k8s-watch-hook.ts

```diff
@@ -61,6 +61,9 @@ const useModelsLoaded = (): boolean => {
   return ref.current;
 };
 
+export const useModelLoaded = (model: string): boolean =>
+  !!useSelector<RootState, K8sKind>(({ k8s }) => k8s.getIn(['RESOURCES', 'models', model]));
```

Comment:
we could check for empty `model` first

Reply:
react hooks shouldn't be called conditionally. eslint will shout at you

Reply:
yes, unless the condition is inside